definitions/types/azure-resource-group.json

```diff
@@ -5,5 +5,9 @@
   "description": "Azure Resource Group",
   "examples": [
     "resource-group-name"
-  ]
+  ],
```

Comment:
Are we using this type anywhere?  I don't see any references in any of the artifacts.  Wondering if we should convert this to a `azure-resource-group-id` and include the full path

Reply:
I think we're using it in the AKS cluster bundle when asking for the DNS zone and resource group name.

Reply:
Ok, let's at least change this to `azure-resource-group-name` then so its clear we are only validating the name, and not path-based ID.